app/scripts/modules/core/src/filterModel/IFilterModel.ts

```diff
@@ -1,23 +1,51 @@
 import { Ng1StateDeclaration, StateParams } from '@uirouter/angularjs';
+import { FilterModelServiceConverters } from 'core/filterModel';
 
 export interface IFilterConfig {
-  model: string;
+  model: keyof ISortFilter;
```

Comment:
woah

Reply:
thanks for the feedback

Reply:
🥈

Reply:
didn't know about `keyof` but it's nice

Reply:
`keyof` tho...
![whoa](https://media.giphy.com/media/MxAJhJ6EEA3zq/giphy.gif)